### ISSUE TITLE
Testers Will Be Needed.

### ISSUE THREAD

**Issue by SpooneyH20** [SpooneyH20/Prohax#1]
Testers are going to be needed throughout this project's development.
In order to qualify for BETA testing you must: 

- Be a reliable person (Who won't leak this exploit until the release is final)

- Have a legit copy of Brunswick Pro Bowling for the 3DS

- Know how basic 3DS homebrew works, and can report back to me with valid bugs/issues

If you think you may be of some assistance to me, please leave your name (or username) and email below.
I'll be sure to contact those i see fit to test when the time comes! 
Thanks, 
   Spooney

**Comment by ThunderRemix** [comment]
Username: Thunder Hawk (GBATemp name)
Email: Not giving it out, sorry. :/

**Comment by SpooneyH20** [comment]
@ThunderRemix I'll setup a discord chat here pretty soon, i'll invite the qualified testers when i'm ready, so it's fine.

**Comment by TheDuckMan64** [comment]
Username: TheDuckMan64 (GBATemp profile name)
Email: I can PM it to you via GBATemp if you need it.

**Comment by TheDuckMan64** [comment]
I've had a blinder, I can't help you sorry :P

**Comment by SpooneyH20** [comment]
@TheDuckMan64 That's alright, and i dont have a GBATemp account due to the bad reception devs get.

**Comment by SpooneyH20** [comment]
Does anyone want to test the crashing [savegame](https://github.com/SpooneyH20/Prohax/commit/5dc7a7ecec35fa74a45e062c6444c960d8da213c) on their copy? It works for me, i just need to know if its working for anyone else.

**Comment by Banamy** [comment]
It crashed for me as well.

**Comment by RealLatias** [comment]
Username: RealLatias (GBATemp Profile Name)
Email: carsonboo@gmail.com

**Comment by SpooneyH20** [comment]
@Banamy that's great! Can i get your FW version and whether your using an actual copy or a CIA please?
@RealLatias Mind testing [crashing_savegame.bin](https://github.com/SpooneyH20/Prohax/blob/master/savegame-files/crashing_savegame.bin?raw=true)
For me?

**Comment by Banamy** [comment]
Im on the latest, cia copy.

**Comment by SpooneyH20** [comment]
Good to know! What did you restore the save with might i ask?

**Comment by ThunderRemix** [comment]
FW: o3DS 9.2.0-20U
CIA or CART: CART
Result: Crash and Reboots

Unless this is exploiting something, I would expect it to crash on every system on any firmware.

**Comment by Banamy** [comment]
I used JK save manager

**Comment by SpooneyH20** [comment]
Thanks for that!

**Comment by cspencer49519** [comment]
I was just tweeting with you, I figured I would bring the discussion here. So I'm just testing to make sure you can cause the game to crash? What's the next step on your end after you verify that the game is in fact crashing on systems? Do you have any method to actually take advantage of the crash yet?

**Comment by RealLatias** [comment]
I'm on a CIA version though.

Thanks for your time,
Carson Sobolewski

**Comment by SpooneyH20** [comment]
@cspencer49519 Yes, if anyone has ANY questions i'd encourage for you to ask in my [discord](https://discord.gg/0xE514XweBC9qYE4) please

**Comment by SpooneyH20** [comment]
@RealLatias Thanks for yours! I changed my mind and you can test with your CIA version, so thanks.

**Comment by cspencer49519** [comment]
Confirming the savegame does crash for me here on an N3DSXL.

**Comment by SpooneyH20** [comment]
Thank you all for the help! If everything goes as planned i should have this running the HBL within the next two weeks.
I'm working on the next savegame, and should have one ready for testing soon.
Thanks again!

**Comment by cspencer49519** [comment]
Also confirming the crash on my O3dsXL

**Comment by Banamy** [comment]
I just bought the real game just now.

**Comment by SpooneyH20** [comment]
I'm working on a functioning 10.7 payload right now. I'm not going to release them all at once because people can update and downgrade through HBL at 10.7.

Expect it by the end of the week/weekend

**Comment by RealLatias** [comment]
Crashes on my 10.7.0-32U New 3DSXL Emunand.

**Comment by SpooneyH20** [comment]
I got the game to crash where it doesent return to the home menu.
This game is now blown wide open and ready to be exploited.
Again, expect it loading the HBL by the end of the week. This new savegame will be uploaded under exploited-savegames

**Comment by SpooneyH20** [comment]
If you'd like to test my semi exploit save you can find it [here](https://github.com/SpooneyH20/Prohax/blob/master/exploited-savegames/exploited_savegame_crash.bin?raw=true)
I'm going to need to know if it works for anyone else

**Comment by Banamy** [comment]
It works. I got a black screen.

**Comment by GalladeGuy** [comment]
Want to know why you're getting a black screen? Its because the CPU is processing his fake instructions as assembly. It reads `ns:s.....fs:USER.ir:rst..am:sys..ptm:sysm.gsp::Lcd.nwm::EXT.news:s..hb:mem0.hb:ndsp.hb:kill.Py,` and thinks that its assembly, so it starts executing a bunch of random stuff, thus making the 3ds crash. So basically, this means pretty much nothing.

**Comment by RealLatias** [comment]
With that semi exploited one, the one Spooney just gave, the game boots like normal. I will try again XD.

Thanks for your time,
Carson Sobolewski

**Comment by RealLatias** [comment]
Nvm, I'm dumb and didn't read the instructions. I get a black screen too :+1:

**Comment by SpooneyH20** [comment]
Nobody needs your bullshit @GalladeGuy i understand what it did, but you didnt.
This crash can load custom ROP code, and you know that because you just forked all of my work to your account.

**Comment by TheDuckMan64** [comment]
Up the Spooney!

**Comment by cspencer49519** [comment]
Are you sure @GalladeGuy forked your code? I don't see that he did. It looks like maybe he cloned it, or even just downloaded the savegame.bin and RE'ed it? Don't get me wrong, not doubting you here, just like everyone else, I'm just eager to see some actual working code.

**Comment by GalladeGuy** [comment]
@SpooneyH20 What I said is bullshit? Feel free to prove me wrong and write some actual ROP code that makes it do anything but crash the 3DS because it isn't even assembly! This is not how it works. Just admit it, you don't have the aptitude to do this! And what's the point of deleting all my comments? You're just butthurt that everyone knows your a fake. But who cares about your reputation? You can just make another dupe. Or just delete this so I know just how much of a pussy you really are. And I told you, the fork was accidental. I was just messing around.

**Comment by Banamy** [comment]
I know next to nothing about all this stuff. But since the crash was moved from startup to in game, doesn't that mean it's controllable in a sense?

**Comment by SpooneyH20** [comment]
To everyone wondering, i was seeing how the game would react to some assembly at all.
By the game crashing this DOES mean it's exploitable. I have said numerous times wait until the end of the week, and i should have a working 10.7 payload. If you'd rather take @GalladeGuy 's way of waiting, and insult my work, call me a "pussy", "butthurt" and say my "reputation sucks" then please do it elsewhere.

**Comment by SpooneyH20** [comment]
@cspencer49519 http://imgur.com/tGnTrHd

**Comment by SpooneyH20** [comment]
@Banamy In a sense, yes. I can somewhat control where the game crashes. But i find that the easier place to do this is when its loading a game like we saw in exploited_savegame_crash.bin
